src/core/style/expressions/torque.js

```diff
@@ -0,0 +1,103 @@
+import Expression from './expression';
+import { implicitCast, DEFAULT } from './utils';
+import { floatDiv, floatMod, now, linear, globalMin, globalMax } from '../functions';
+import Property from './property';
+
+const DEFAULT_FADE = 0.15;
+
+/**
+ * Create a Torque FadeIn/FadeOut configuration
+ *
+ * @param {carto.style.expression.expression|Number} param1 expression of type float or Number
+ * @param {carto.style.expression.expression|Number} param2 expression of type float or Number
+ * @return {carto.style.expressions.fade}
+ *
+ * @example `
+// fadeIn of 0.1 seconds, fadeOut of 0.3 seconds
+filter:       torque($day, 40, fade(0.1, 0.3))
+
+// fadeIn and fadeOut of 0.5 seconds
+filter:       torque($day, 40, fade(0.5))
+
+`
+ *
+ * @memberof carto.style.expressions
+ * @name fade
+ * @function
+*/
+export class Fade extends Expression {
```

Comment:
I think `Fade` should be in its own file

Reply:
I haven't moved it since, at least right now, `Fade` is entirely made to be used with `torque`

Reply:
I haven't moved it since, at least right now, `Fade` is entirely made to be used with `torque`